scripts/clean.mjs

```diff
@@ -0,0 +1,79 @@
+import FastGlob from "fast-glob";
+import { existsSync, lstatSync, readdirSync, rmdirSync, unlinkSync } from "fs";
+import { join } from "path";
+
+/**
+ * This script is used to clean up build artifacts.
+ * - Removes all node_modules folders if `--reset`-argv is passed
+ */
+const globPatterns = [
+  "./@navikt/**/lib",
+  "./@navikt/**/esm",
+  "./@navikt/**/cjs",
+  "./@navikt/aksel-icons/src",
+  "**/playwright-report",
+  "**/.next",
+  "**/dist",
+];
+
+console.group("Cleaning up build artifacts");
+
+for (const globPattern of globPatterns) {
+  const folders = FastGlob.sync(globPattern, {
+    dot: true,
+    onlyDirectories: true,
+    ignore: ["**/node_modules"],
+  });
+  folders
+    .filter((folder) => !folder.includes("node_modules"))
```

Comment:
Is this needed in addition to `ignore: ["**/node_modules"]`?

Reply:
Nope